### ISSUE TITLE
py4vasp.exceptions.exceptions.FileAccessError: 

### ISSUE THREAD

**Issue by ftry** [vasp-dev/py4vasp#50]
After Calculation，why not produced the h5 file?  I use the latest version vasp! but produced xml file!
the py4vsp just reads h5 file?

**Comment by ftry** [comment]
I found it that l complied vasp without hdf5!
So ,if who wan to use py4vasp ,he must complie vasp with hdf5!

**Comment by martin-schlipf** [comment]
Yes, to use py4vasp, you need to compile VASP with HDF5 support. The XML output is still provided for legacy reasons, but all new developments should built on the HDF5 files.
